content/minutes/cab/2021-10-20-minutes.md

```diff
@@ -0,0 +1,27 @@
+---
+date: "2021-10-20T00:00:00+00:00"
+title: "Oct 20, 2021 Advisory Board Meeting"
+tags: ["Open Force Field Consortium", "Advisory Board", "minutes"]
+categories: ["Advisory Board Meeting Minutes"]
+draft: false
+description: "Meeting minutes"
+weight: 10
+author: "Diego Nolasco"
+---
+
+The [Open Force Field Consortium Advisory Board](https://openforcefield.org/about/organization/#open-force-field-consortium) met on October 20, 2021.
+The minutes are summarized as follows:
+
+#### Espaloma as a potential next-generation OpenFF Technology
+
+* D. Mobley presented [Espaloma](https://drive.google.com/file/d/1dm9a7rmO-tAUjuSQGHjAg8JrjVu3aGPC/view) as a potential next-generation technology for the OpenFF force field, inviting the advisors for deeper discussions on our [#espaloma](https://join.slack.com/share/enQtMzU3NDA4NjQ2MDY5Mi1kYWJkMjUzNmY0NGVlMDcwNWI5ZDk1ZDRlMjYzZjdjNjczMTcxOGFlNGJjN2U0YTI3MzI0MjM4ODNiNjkyOGNl) channel on Slack. The [software](https://github.com/choderalab/espaloma) has been produced by J. Chodera's lab, and partially supported by NIH funding.
```

Comment:
Slack users cost us $1 per month, and the link here would let anyone join and could balloon our costs.

```suggestion
* D. Mobley presented [Espaloma](https://drive.google.com/file/d/1dm9a7rmO-tAUjuSQGHjAg8JrjVu3aGPC/view) as a potential next-generation technology for the OpenFF force field, inviting the advisors for deeper discussions on our #espaloma channel on Slack. The [software](https://github.com/choderalab/espaloma) has been produced by J. Chodera's lab, and partially supported by NIH funding.
```